### ISSUE TITLE
You can enter in your own wool room (Some maps)

### ISSUE THREAD

**Issue by Zenzoh** [twizmwazin/CardinalPGM#492]
![image](https://cloud.githubusercontent.com/assets/11640813/6889486/45967c8c-d654-11e4-8642-5234c0e007b5.png)

XML: http://pastebin.com/kb8yazNA

Version: *9f9f043*

EDIT: This doesn't happen in map proto 1.3.0

**Comment by AngelCasillas01** [comment]
Appears to be a `<regions>` problem, maybe a bug in c3dfc8cd27

**Comment by Carlos934** [comment]
@AngelCasillas01 That does not touch anything of <regions>

**Comment by AngelCasillas01** [comment]
@Carlos934 Yes it does, they modify and check liquid regions, so they have added `BlockPlaceEvent` isn't nothing to do but something there maybe has bugged something about this

**Comment by AngelCasillas01** [comment]
@Carlos934 Try compiling Cardinal for lastest version and try it again, make sure to post the new log :)

**Comment by twizmwazin** [comment]
@AngelCasillas01 The commit version is what we want, so a link to the commit is perfectly fine.

**Comment by Carlos934** [comment]
@twizmwazin ¿Al final cual es el error?

**Comment by AngelCasillas01** [comment]
@twizmwazin es "were is the bug on" estas diciendo que es el bug ;)

**Comment by Carlos934** [comment]
@AngelCasillas01 I don't have a doctorate in English, I speak Spanish

**Comment by AngelCasillas01** [comment]
@Carlos934 I'm just correcting you, i speak spanish too

**Comment by Carlos934** [comment]
@AngelCasillas01 Me refería a.. ¿Cuál es el error?

No me refería a ¿Dónde está el error?

Sorry for the Spanish.
@AngelCasillas01 Me refería a.. ¿Cuál es el error?

No me refería a ¿Dónde está el error?

Sorry for the Spanish.

**Comment by AngelCasillas01** [comment]
@Carlos934 por eso, pero no se traduce inmediatmente como lo hace google, lo haces con el common sense de la gramatica en ingles, puedes usar lo que te dije o puedes dejarlo haci pero como usted quiera :)

**Comment by TiTo-418** [comment]
@AngelCasillas01 @Carlos934 Traten de mantener el el problema aqui, pueden discutir eso en privado.
Try to keep the issue here, you can discuss that in private

**Comment by Carlos934** [comment]
Último, no tengo idea de como se dice cual, y no uso traductor :'D
Esto se termina, mejor hay que concentrarnos en CardinalPGM.

Last, I don't have idea how say "cual", and i don't use translator :'D
This has finished, we must concentrate in Cardinal PGM.

**Comment by SungMatt** [comment]
Not sure what you mean by "this has finished", just wanted to let you know that Reshif's public download of GDIII is a bit different from the one on overcast, so a direct porting of XML will not work.

**Comment by twizmwazin** [comment]
@SungMatt is correct, I remember having to nudge the map an mcedit. However, there are still a few lingering region issues in Cardinal that need to be worked out.

**Comment by Carlos934** [comment]
@SungMatt is a map downloaded with World Downloader... do not tell anyone ...

**Comment by CaptainElliott** [comment]
@Carlos934 You say that with a map developer helping develop the plugin :p

**Comment by Carlos934** [comment]
@CaptainElliott I mean that the map is correct with xml

**Comment by elespawol** [comment]
Dont worry carlos, its LEGAL on oc.tc,
But u have a rule: don't distribute with out author's permission 😉
El dia 30/03/2015 17.34, "Carlos934" <notifications@github.com> va escriure:

**Comment by iEli2tyree011** [comment]
Closing as this seems to be an issue between the XML and the map; they don't match.

**Comment by Carlos934** [comment]
Reopen, the map fits perfectly

**Comment by TiTo-418** [comment]
Try with this xml: view-source:https://maps.oc.tc/Other/Tournament/Guardians%20of%20the%20Wool/Golden%20Drought%20III/map.xml

**Comment by Carlos934** [comment]
@TiTo-418 I use that

**Comment by twizmwazin** [comment]
Are you guys using the map as it is on OCN or are you using Reshif's download?

**Comment by iEli2tyree011** [comment]
@twizmwazin The problem is that parent filters are not implemented, the problem isn't a mismatched map and XML.

**Comment by TiTo-418** [comment]
Reshif's download, with both XML (Reshif dowload and maps.oc.tc)

**Comment by Eudaldca** [comment]
@CaptainElliott You however cannot enter your own wool room, so that *is* fixed. It is also not an issue of the map being incorrectly positioned, as it happens in both lanes equally.

**Comment by TiTo-418** [comment]
Please, reopen this
Now you can not enter in your own wools room, but you can place blocks on it.

<a href="http://imgur.com/GaWP2Ni"><img src="http://i.imgur.com/GaWP2Ni.png" title="source: imgur.com" /></a>

**Comment by dentmaged** [comment]
@twizmwazin Shouldn't this have a "bug" tag?

**Comment by TiTo-418** [comment]
Maybe this is an XML problem, im not sure

**Comment by Eudaldca** [comment]
For the ones subscribed to this issue, I guess this will interest you. I modified the xml by adding 2 more applied filters so you cannot place blocks in your own wool room. Feel free to comment my gist if you have a better way of doing it, but I think it's fine the way I've done it. [_Link_](https://gist.github.com/1a374b971212e1fa1640)

**Comment by TiTo-418** [comment]
@iEli2tyree011 @twizmwazin Could you change the issue's title?

**Comment by Alfalik** [comment]
Pretty sure this issue was fixed on https://github.com/twizmwazin/CardinalPGM/commit/09d13e740b10751ba640060a36e664618c803f1c
